.travis.yml

```diff
@@ -20,32 +20,9 @@ before_install:
   - python -m compileall -q -f .
 install:
   - pip install --upgrade pip # FIXME remove once Travis support pip7 by default
-  - CFLAGS="-O0" pip install -r requirements/travis.txt
-  - pip install "Django>=$DJANGO_VERSION,<"$(echo $(echo $DJANGO_VERSION | cut -d"." -f1).$(( $(echo $DJANGO_VERSION | cut -d"." -f2) + 1 )) )
-  - pip install .
-  - pip freeze  # Print all installed versions for reference.
-before_script:
-  - if [[ $DATABASE_BACKEND == 'postgres' ]]; then psql -c 'create database pootle;' -U postgres; fi
-  - if [[ $DATABASE_BACKEND == 'mysql_innodb' ]]; then mysql -e 'create database pootle CHARACTER SET utf8 COLLATE utf8_general_ci;'; fi
-  - if [[ $DATABASE_BACKEND == 'mysql_innodb' ]]; then mysql -e "SET GLOBAL wait_timeout = 36000;"; fi
+  - pip install tox
```

Comment:
There is already one.  Likely travis.txt can go away as its done differently in tox